diff --git a/Android.bp b/Android.bp
index b941928..3cfbf28 100644
--- a/Android.bp
+++ b/Android.bp
@@ -16,11 +16,8 @@ rust_defaults {
     ],
     rustlibs: [
         "libfutures",
-        "libsyn",
         "libthiserror",
-        "libtrybuild",
     ],
-    proc_macros: ["librustversion"],
 }
 
 rust_test_host {
@@ -52,11 +49,8 @@ rust_defaults {
     rustlibs: [
         "libanyhow",
         "libfutures",
-        "libsyn",
         "libthiserror",
-        "libtrybuild",
     ],
-    proc_macros: ["librustversion"],
 }
 
 rust_test_host {
